core-services/release-controller/_releases/release-ocp-4.17-multi.json

```diff
@@ -53,49 +59,137 @@
         }
       }
     },
-    "multi-e2e-azure-ovn": {
+    "e2e-azure-ovn-multi-x-ax": {
       "optional": true,
       "prowJob": {
-        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-azure-ovn-heterogeneous"
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-azure-ovn-multi-x-ax"
       }
     },
-    "multi-e2e-gcp-ovn": {
+    "e2e-gcp-ovn-multi-x-ax": {
       "optional": true,
       "prowJob": {
-        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-gcp-ovn-heterogeneous"
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-gcp-ovn-multi-x-ax"
       }
     },
-    "multi-e2e-ovn": {
-      "maxRetries": 2,
+    "e2e-nightly-upgrade-gcp-ovn-multi-x-ax": {
+      "optional": true,
+      "prowJob": {
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-upgrade-gcp-ovn-multi-x-ax"
+      },
+      "upgrade": true,
+      "upgradeFrom": "Previous"
+    },
+    "e2e-minor-upgrade-gcp-ovn-multi-x-ax": {
+      "optional": true,
+      "prowJob": {
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-upgrade-from-stable-4.16-ocp-e2e-upgrade-gcp-ovn-multi-x-ax"
+      },
+      "upgrade": true,
+      "upgradeFromRelease": {
+        "candidate": {
+          "stream": "nightly-multi",
+          "version": "4.16"
+        }
+      }
+    },
+    "e2e-serial-aws-ovn-multi-x-ax": {
+      "optional": true,
+      "prowJob": {
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-serial-aws-ovn-multi-x-ax"
+      }
+    },
+    "e2e-aws-ovn-multi-a-a": {
+      "optional": true,
+      "maxRetries": 3,
+      "prowJob": {
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-multi-a-a"
+      }
+    },
+    "e2e-ovn-serial-aws-multi-a-a": {
+      "optional": true,
+      "prowJob": {
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-ovn-serial-aws-multi-a-a"
+      }
+    },
+    "e2e-aws-ovn-sno-multi-a-a": {
+      "optional": true,
       "prowJob": {
-        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-heterogeneous"
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-sno-multi-a-a"
       }
     },
-    "multi-gcp-ovn-upgrade": {
+    "e2e-aws-ovn-techpreview-multi-a-a": {
       "optional": true,
       "prowJob": {
-        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-upgrade-gcp-ovn-heterogeneous"
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-techpreview-multi-a-a"
+      }
+    },
+    "e2e-aws-ovn-techpreview-serial-multi-a-a": {
```

Comment:
This one looks like it's breaking naming convention, but I'm not sure we care about release controller naming conventions matching our job name spec.